includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -2720,6 +2721,15 @@ public function enable() {
 		$this->update_option( 'enabled', 'yes' );
 	}
 
+	/**
+	 * Returns the default list of enabled payment method types for UPE.
+	 *
+	 * @return string[]
+	 */
+	public function get_default_upe_enabled_payment_method_ids(): array {
```

Comment:
IMO. The `default` name is not really correct, and it's confusing when reading other functions using this method. 

The better name may be exactly what it does, something like: `filter_card_payment_method_id_enabled`? 

Also, we may narrow down the scope of this function to `protected` as it's being used in the inheritance class `UPE_Payment_Gateway` too but nowhere else. 

With the current confusing way we're currently constructing these classes: `CC_Payment_Gateway` (empty code inside) and `UPE_Payment_Gateway` are child classes of `WC_Payment_Gateway_WCPay`, I think we may want to make this function `final` too, i.e. not allowing to modify the function in `UPE_Payment_Gateway`.

Reply:
Renamed to be less confusing.